docs/reference/oras-auth-provider.md

```diff
@@ -143,69 +182,49 @@ EOF
+az keyvault certificate import \
+  --vault-name ${KEYVAULT_NAME} \
+  -n <Certificate Name> \
+  -f /path/to/certificate
+
+# Grant permission to access the certificate.
+az keyvault set-policy --name ${KEYVAULT_NAME} \
+  --certificate-permissions get \
+  --object-id ${IDENTITY_OBJECT_ID}
 ```
-### 3. Azure Managed Identity
-
-Ratify pulls artifacts from a private Azure Container Registry using User-assigned Managed Identity in an Azure Kubernetes Service cluster.  
-Currently, it does not support cross tenants for MSI and ACR.
-
-#### User steps to set up Managed Identity with AKS and ACR:
-
-1. Create ACR
-2. Create AKS cluster
-3. Configure managed identity on the AKS managed VMSS.  
-   a. Directly using kubelet identity:
-      If your AKS cluster has already run `az aks update --attach-acr` to ensure access to the private ACR, kubelet identity will directly has the permission to pull artifacts.
-      ```shell
-      export IDENTITY_CLIENT_ID=$(az aks show -g ${AKS_CLUSTER_NAME} -n ${RESOURCE_GROUP} --query "identityProfile.kubeletidentity.clientId")
-      ```
-   b. Bring your own user-assigned managed identity:
-      ```shell
-      # Create one identity
-      az identity create --name ${IDENTITY_NAME} --resource-group ${RESOURCE_GROUP}
-      export IDENTITY_CLIENT_ID=$(az identity show --name myIdentity --resource-group ${RESOURCE_GROUP} --query 'clientId' -o tsv)
-      export IDENTITY_PRINCIPAL_ID=$(az identity show --name myIdentity --resource-group ${RESOURCE_GROUP} --query 'principalId' -o tsv)
-      export IDENTITY_RESOURCE_ID=$(az identity show --name myIdentity --resource-group ${RESOURCE_GROUP} --query 'id' -o tsv)
-
-      # Assign identity ACR pull permission to ACR
-      az role assignment create --assignee ${IDENTITY_PRINCIPAL_ID} --role "Acrpull" --scope "/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP}/providers/Microsoft.ContainerRegistry/registries/${ACR_NAME}"
-
-      # Configure MSI on all AKS agent pools
-      node_resource_group=$(az aks show -g ${AKS_CLUSTER_NAME} -n ${RESOURCE_GROUP} --query "nodeResourceGroup")
-      for vm in ${az vmss list -g $nodeResourceGroup --query "[].name" -o tsv}
-      do
-        az vmss identity assign -g $node_resource_group -n $vm --identities ${IDENTITY_RESOURCE_ID}
-      done
-      ```
-
-4. Deploy ratify helm chart
+
+8. Deploy from local helm chart. Follow the [Quick Start](https://github.com/deislabs/ratify/blob/main/README.md#quick-start) to deploy Gatekeeper and Ratify.
+
+Notes: add below options while installing Ratify
 ```shell
-export TENANT_ID={{ Tenant Id }}
-helm install ratify \
-    ratify/ratify --atomic \
-    --set oras.authProviders.azureManagedIdentityEnabled=true \
-    --set azureManagedIdentity.clientId=${IDENTITY_CLIENT_ID} \
-    --set azureManagedIdentity.tenantId=${TENANT_ID}
+--set azureWorkloadIdentity.clientId=${IDENTITY_CLIENT_ID} \
```

Comment:
should we just include the `helm install` part of the command here too?

Reply:
@FeynmanZhou what's your opinion on this?

Reply:
It's not necessary to mention `helm install` in this doc. We can include it in the [Quick Start](https://github.com/deislabs/ratify/blob/main/README.md#quick-start) and related docs, just mentioned helm is a prerequisite.